### PR TITLE
Add hello world with graphql

### PR DESCRIPTION
**Pull request by ryonryon**
@axross Can you check?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
     "dev": "ts-node --transpile-only ./node_modules/micro-dev/bin/micro-dev.js -l tcp://0.0.0.0:$PORT ./src/main.ts"
   },
   "dependencies": {
+    "@types/graphql": "^14.0.7",
```
Comment (axross) [inline package.json]: This is supposed to be in `devDependencies`.

File: package.json [hunk 2]
```diff
@@ -18,6 +18,9 @@
...
+    "graphql": "^14.1.1",
+    "http": "0.0.0",
```
Comment (axross) [inline package.json]: Where do you use `http` package in?

File: src/main.ts [hunk 3]
```diff
@@ -32,6 +45,13 @@ async function main(request: IncomingMessage, response: ServerResponse) {
     });
   }
 
+  if (request.method === "GET" && pathname === "/graphql") {
+    if (searchParams.has("query")) {
```
Comment (axross) [inline src/main.ts]: You should concat these three conditions into one if statement.

**Review by axross — CHANGES_REQUESTED** [review]
LGTM, but there's few thing to be fixed.

**Review by ryonryon — COMMENTED** [review]
@axross I changed the result of your review. Can you check it again?

**Comment by ryonryon** [comment]
@axross test passed now. Can you check again?

**Comment by axross** [comment]
Test has still been failed. Maybe you missed to close an ad-hoc server in the test code.

**Comment by ryonryon** [comment]
@axross  I got it!

File: src/__tests__/main-test.ts [hunk 4]
```diff
@@ -4,48 +4,17 @@ import listen = require("test-listen");
 import { URL } from "url";
 import handler = require("../main");
 
-describe("GET /", () => {
-  test("responds with value that is given at `message` query parameter", async () => {
+describe("GET /graphql", () => {
+  test("responds graphql data", async () => {
```
Comment (axross) [inline src/__tests__/main-test.ts:8]: I think this is much better

```suggestion
  test("responds data through graphql", async () => {
```

File: src/main.ts [hunk 5]
```diff
@@ -1,8 +1,19 @@
...
+const schema = buildSchema(`
+  type Query {
+    hello: String
+  }
+`);
+
+const root = {
+  hello: () => {
+    return 'Hello world!';
+  },
```
Comment (axross) [inline src/main.ts:15]: You can omit block `{}` and `return` statement.

```ts
  hello: () => 'Hello world!',
```

**Review by axross — APPROVED** [review]
LGTM

**Pull request merged**